spec/classes/web_spec.rb

```diff
@@ -53,7 +55,7 @@
           it { is_expected.not_to contain_selboolean('httpd_can_connect_zabbix') }
         end
 
-        %w[4.4 5.0].each do |zabbix_version|
+        %w[4.0 5.0].each do |zabbix_version|
           describe "with database_type as postgresql and zabbix_version #{zabbix_version}" do
             next if zabbix_version == '5.0' and %w[Debian Ubuntu].include?(facts[:osfamily]) and
```

Comment:
I think I'm doing the same here as you are doing at beginning of the file. Maybe this can be removed then?

Reply:
https://github.com/voxpupuli/puppet-zabbix/pull/751/files/fa5f23d200347590a247724b01850f24908f63e4#diff-82ccb785b8e760dc28ab7067a48ebd387c93d71c401fbebb9256bfa38117014aR17

Reply:
cleaned it up